hazelcast-enterprise/Dockerfile

```diff
@@ -89,6 +89,7 @@ CMD ["bash", "-c", "set -euo pipefail \
       && echo \"# CLASSPATH=${CLASSPATH}\" \
       && echo \"# starting now....\" \
       && echo \"########################################\" \
+      && if [[ \"x${HZ_LICENSE_KEY}\" == \"x\" && -v HZ_LICENCE_KEY ]]; then echo \"\nWARNING: Misspelled License Key variable, define HZ_LICENSE_KEY instead of HZ_LICENCE_KEY\n\"; fi \
```

Comment:
Lets just accept both... why warning with that? 😄

Reply:
@mesutcelik , I like the idea. Changed to accepting both versions. PTAL.